packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -15,6 +15,7 @@ export class DdSdkReactNativeConfiguration {
     public nativeCrashReportEnabled: boolean = false;
     public sampleRate: number = 100.0;
     public site: string = "US"
+    public verbosity: string|undefined = undefined
```

Comment:
I think it is useful to give customer possible values, because even for warning it is not clear should it be `warn` or `warning`. Maybe it makes sense to even use enum.